manifest: Update upmerged respositories (#12619)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

      * The ``CONFIG_CHIP_SED_ACTIVE_INTERVAL`` Kconfig option was renamed to :kconfig:option:`CONFIG_CHIP_ICD_FAST_POLLING_INTERVAL`.
      * The ``CONFIG_CHIP_SED_ACTIVE_THRESHOLD`` Kconfig option was renamed to :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_THRESHOLD`.

* For samples using Wi-Fi features:

>   * A few Kconfig options related to scan operations have been removed in the current release. If your application uses them,
>     they will need to be updated to remove the dependency on these:

Reviewer: ```suggestion
  * A few Kconfig options related to scan operations have been removed in the current release. If your application uses scan operations, they need to be updated to remove the dependency on the following options:
```

Reviewer 2: Updates
